### ISSUE TITLE
Graph line characters are displaying as blocks of dots

### ISSUE THREAD

**Issue by eliliam** [ClementTsang/bottom#337]
### Question

When using the Kitty terminal, all of the graphs are made up of blocks with varying size dots, instead of a single dotted line as expected. I tested against the xfce4-terminal and it rendered properly there. I am using IBM Plex Mono as a font, with stock kitty configs. Any guidance would be greatly appreciated! 

Here is how it looks in kitty:
![btm](https://user-images.githubusercontent.com/14301439/100946236-2db2f480-34c0-11eb-9f32-41202a8fe6e2.png)

#### Platform
Operating System: Arch Linux
Terminal: Kitty
Shell: zsh
Bottom version: 0.5.3

**Comment by ClementTsang** [comment]
It comes down to how your font/terminal displays braille, AFAIK.

See https://github.com/kovidgoyal/kitty/issues/1623, this might be helpful.

**Comment by eliliam** [comment]
That was it! I was missing a braille font, so after installing ttf-ubraille from the AUR everything looks fantastic!

**Comment by ClementTsang** [comment]
Nice, good to hear it worked out!

**Comment by jcaesar** [comment]
Arch Linux + Urxvt here. Uninstalling `gnu-free-fonts` only brought me more boxes, and ttf-ubraille does not seem to contain all braille characters (Try e.g. `echo -e \u2810  \u2840`). I ended up looking for appropriate fonts with [this script](https://stackoverflow.com/a/19438403/401059), and then tweaking my `~/.Xdefaults`
```
URxvt*font:     xft:Source Code Pro Semibold:size=10,xft:DejaVu Sans
! Source Code Pro Semibold doesn't contain the braille characters, but looks nice, DejaVu from ttf-dejavu contains them. Try DejaVu only, if in doubt. Iosevka Nerd Font from ttf-iosevka-nerd may be an alternative.
```

**Comment by heyakyra** [comment]
I am confused. On the latest Fedora Silverblue, I installed `dejavu-fonts-all` `google-droid-fonts-all` and `fontawesome5-fonts-all` (didn't uninstall anything). When I tried to use each of their `mono` variants in my `alacritty` config file at `~/.config/alacritty/alacritty.yml`, that worked! 

So then I purged that file and tried setting those fonts in `gnome-tweak-tool` which also worked!

But then I set it back to Source Code Pro…and it still worked? Why would merely having a font installed make a difference, even when it isn't the one being used?
